src/dreammaker/builtins.rs

```diff
@@ -1060,6 +1088,35 @@ pub fn register_builtins(tree: &mut ObjectTree) {
 
         regex/proc/Find_char(text, start, end);
         regex/proc/Replace_char(text, rep, start, end);
+
+        // 514 stuff
+
+        generator;
```

Comment:
it'd be nice to have an enum for the various `type`s, and the applicable defines for rand being an enum as well

Reply:
not sure how to handle the operator overloads for this either

Reply:
the existence of operator overloads for builtins isn't currently described by SpacemanDMM